Source/WebKit/WebProcess/Plugins/PDF/PDFPlugin.mm

```diff
@@ -652,10 +652,7 @@ static void getAllScriptsInPDFDocument(CGPDFDocumentRef pdfDocument, Vector<Reta
 #endif
     , m_identifier(PDFPluginIdentifier::generate())
 {
-    // FIXME: <rdar://101787977> Replace this with SPI once we get it from PDFKit
-    [[NSUserDefaults standardUserDefaults] setBool:NO forKey:@"PDFKit2_UseIOSurfaceForTiles"];
-    [[NSUserDefaults standardUserDefaults] setBool:NO forKey:@"PDFKit2_UseWhippet"];
-
+    m_pdfLayerController.get().useIOSurfaceForTiles = false;
```

Comment:
Can we move this logic beneath the `document` variable assignment, grouping it with the rest of the PDF logic?

Reply:
Fixed.